install_pks.sh

```diff
@@ -22,10 +22,16 @@ export UAA_ADMIN_PASSWORD=$(echo $TOOLSMITH_ENV | base64 --decode | jq -r .pks_a
 
 pks login -a api.pks.${TS_G_ENV}.cf-app.com -u admin -p ${UAA_ADMIN_PASSWORD} -k
 
-# TODO uncomment if we use a PKS cluster per job
-# pks create-cluster ${TS_G_ENV}-${CLUSTER_NAME} --external-hostname ${CLUSTER_NAME}.${TS_G_ENV}.cf-app.com --plan small --wait
+gcloud compute addresses create ${TS_G_ENV}-${CLUSTER_NAME}-ip --region=${CLOUDSDK_COMPUTE_REGION}
+export LB_IP=`gcloud compute addresses list --filter="name=(${TS_G_ENV}-${CLUSTER_NAME}-ip)" --format=json | jq -r .[0].address`
+
+pks create-cluster ${TS_G_ENV}-${CLUSTER_NAME} --external-hostname ${LB_IP} --plan large --wait
```

Comment:
I assume you're skipping setting up dns, by using the ip directly here instead of a hostname, nice.